test/index.js

```diff
@@ -335,16 +358,6 @@ describe('client JavaScript output', function () {
       return done();
     });
   });
-  it('--name-after-file _InPuTwIthWEiRdNaMME.pug', function (done) {
```

Comment:
Okay can you keep the test but maybe change the `_` to a hyphen or something so the coverage is kept?